Undeclared dependency in styles package

### Expected Behaviour
I can use Spectrum Web Components with Yarn 2 without seeing dependency errors.

### Actual Behaviour
.yarn/cache/@spectrum-web-components-styles-npm-0.7.0-769d2cfc5c-b606d82381.zip/node_modules/@spectrum-web-components/styles/src/spectrum-base.css.js
Module not found: @spectrum-web-components/styles tried to access @spectrum-web-components/base, but it isn't declared in its dependencies; this makes the require call ambiguous and unsound.

#### Additional info

You can see this in the **generated** code here: https://unpkg.com/browse/@spectrum-web-components/styles@0.7.0/src/spectrum-base.css.js

There isn't actually any reference to '@spectrum-web-components/base' in the source code for the styles package though,  so there's something about the tooling that adds it.

--- a/packages/styles/package.json
+++ b/packages/styles/package.json
@@ -42,6 +42,7 @@
     "author": "",
     "license": "Apache-2.0",
     "devDependencies": {
+        "@spectrum-web-components/base": "^0.2.0",
         "@spectrum-css/commons": "^3.0.0",
         "@spectrum-css/typography": "^3.0.0-beta.1",
         "@spectrum-css/vars": "^3.0.0-beta.2"

